roles/kubernetes/preinstall/tasks/main.yml

```diff
@@ -178,7 +178,13 @@
 - include: etchosts.yml
   tags: [bootstrap-os, etchosts]
 
-- include: growpart-centos-7.yml
-  when: ansible_distribution in ["CentOS","RedHat"] and
+- name: Check if we are running inside a Azure VM
```

Comment:
I'm not a big fan of this change of checking for Azure. Why not check cloud_provider is defined and cloud_provider == 'azure' like above?